stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -1,12 +1,25 @@
 # Stubs for multiprocessing
 
-from typing import Any
+from typing import Any, Callable, Iterable, Mapping
 
 class Lock(): ...
-class Process(): ...
+class Process():
+    # TODO: set type of group to None
+    def __init__(self,
+                 group: Any = ...,
+                 target: Callable = ...,
+                 name: str = ...,
+                 args: Iterable[Any] = ...,
+                 kwargs: Mapping[Any, Any] = ...,
+                 daemon: bool = ...) -> None: ...
+
```

Comment:
I prefer no blank lines between methods in stub files -- the denser layout is actually more readable since most methods are one line only.